mysql/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://www.mysql.com/about/legal/) for the software contained in this image.
```

Comment:
@ltangvald is this an appropriate place to link for MySQL license information? (it's non-obvious, but seems to be the best I could find in a quick search)